packages/studio-base/src/AppSetting.ts

```diff
@@ -16,7 +16,6 @@ export enum AppSetting {
   ENABLE_NEW_TOPNAV = "enableNewTopNav",
 
   // Privacy
-  TELEMETRY_ENABLED = "telemetry.telemetryEnabled",
```

Comment:
Can you explain why did you decide to remove telemetry?

Reply:
I think it was a mistake, as I thought it was not used anymore. 
I reverted it.